project/DistributionPackage.scala

```diff
@@ -203,6 +203,7 @@ object DistributionPackage {
           val command = Seq(
             Platform.executableFileName(ensoExecutable.toString),
             "--no-compile-dependencies",
+            "--no-global-cache",
```

Comment:
`--no-global-cache` in this case means "distribution cache". "Global" in fact means "per user"... If Enso runtime was installed in the system and shared between multiple users, then the "distribution cache" would be more "global" than our current "global". The terminology is a mess.